src/pages/studyView/addChartButton/geneLevelSelection/styles.module.scss

```diff
@@ -0,0 +1,35 @@
+.body {
+    display: flex;
+    flex-direction: column;
+    textarea {
```

Comment:
this introduces new style/behavior that won't present in other places where OQLTextArea is used.  is it necessary?  Should it be part of component?  Just want to keep things consistnet.

Reply:
this won't be required. I'll clean it up